cocos/renderer/CCTextureCache.h

```diff
@@ -204,6 +204,12 @@ class CC_DLL TextureCache : public Ref
      */
     const std::string getTextureFilePath(Texture2D* texture)const;
 
+    void setDirty(bool dirty) { _dirty = dirty; }
+    bool isDirty() const { return _dirty; }
+
+    void renameTextureWithKey(std::string srcName, std::string dstName);
```

Comment:
`std::string srcName` -> `const std::string& srcName`
`std::string dstName` -> `const std::string& dstName`

And please add document for new functions.